### ISSUE TITLE
Added PARTICIPLE english conjugation types

### ISSUE THREAD

**Issue by ncoder** [RosaeNLG/rosaenlg#84]
I found myself needing the "getIngPart()" form for a verb, and also the "getPastPart()" output to make a sentence.

I don't really know what I'm doing, but using this:
https://conjugator.reverso.net/conjugation-english-verb-take.html
it seems that these verb forms have a name, and that they are called PARTICIPLE

So i added these tenses for myself.

Is this useful? Is this correct?

```diff
diff --git a/node_modules/english-verbs-helper/dist/index.js b/node_modules/english-verbs-helper/dist/index.js
index 98b8373..e83eb90 100644
--- a/node_modules/english-verbs-helper/dist/index.js
+++ b/node_modules/english-verbs-helper/dist/index.js
@@ -27,6 +27,9 @@ const tenses = [
     'PERFECT_PROGRESSIVE_PAST',
     'PERFECT_PROGRESSIVE_PRESENT',
     'PERFECT_PROGRESSIVE_FUTURE',
+    // PARTICIPLE
+    'PARTICIPLE_PRESENT',
+    'PARTICIPLE_PAST'
 ];
 const modals = ['can', 'could', 'may', 'might', 'must', 'shall', 'should', 'will', 'would'];
 // helpers
@@ -284,6 +287,10 @@ function getConjugation(verbsInfo, verb, tense, number, extraParams) {
             return getPerfectProgressivePresent(getIngPart(getVerbInfo(verbsInfo, verb), verb), number);
         case 'PERFECT_PROGRESSIVE_FUTURE':
             return getPerfectProgressiveFuture(getIngPart(getVerbInfo(verbsInfo, verb), verb));
+        case 'PARTICIPLE_PRESENT':
+            return getIngPart(getVerbInfo(verbsInfo, verb), verb);
+        case 'PARTICIPLE_PAST':
+            return getPastPart(getVerbInfo(verbsInfo, verb), verb);
     }
 }
 exports.getConjugation = getConjugation;
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by ludans** [comment]
hi @ncoder

I agree, this might be useful, and should be added.
Names PARTICIPLE_PRESENT and PARTICIPLE_PAST look ok.

Are you willing to make a PR with that?
- update packages\english-verbs-helper\lib\index.ts
- update the doc packages\rosaenlg-doc\doc\modules\mixins_ref\pages\verbs_english.adoc
- add some tests packages\english-verbs-helper\test\test.js

Tell me.

**Comment by ncoder** [comment]
Yes. please let me pull this together into a proper PR.

**Comment by ludans** [comment]
will be available in > 3.0.4
